src/lidar/lidar_metadata_in_db.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct  5 16:34:48 2021.
+
+@author: pkh35
+"""
+
+import geoapis.lidar
+import json
+import geopandas as gpd
+import os
+import zipfile
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String
+from geoalchemy2 import Geometry
+from sqlalchemy.orm import sessionmaker
+from src.digitaltwin import setup_environment
+
+Base = declarative_base()
+
+
+class Lidar(Base):
+    """Create lidar table in the database."""
+
+    __tablename__ = 'lidar'
+    unique_id = Column(Integer, primary_key=True, autoincrement=True)
+    filepath = Column(String)
+    Filename = Column(String)
+    filename_no_format = Column(String)
+    geometry = Column(Geometry('POLYGON'))
+
+
+engine = setup_environment.get_database()
+Lidar.__table__.create(bind=engine, checkfirst=True)
+
+
+def get_files(filetype, folder):
+    """To get the path of the downloaded point cloud files."""
+    file_list = []
+    files_path = []
+    for (paths, dirs, files) in os.walk(folder):
+        for file in files:
+            if file.endswith(filetype):
+                file_list.append(os.path.join(paths, file))
+    for filepath in file_list:
+        filepath = filepath.replace(os.sep, '/')
+        filepath = "'{}'".format(filepath)
+        filepath = filepath.replace("'", "")
+        files_path.append(filepath)
```

Comment:
Some comments on what you are doing here would be good. I can see you replace the separator but not sure what the next bit is

Reply:
It seems like you add ' to the ends and then remove it.

Another way to remove only from the ends is `str.strip`